docker/build_docker.sh

```diff
@@ -233,11 +233,13 @@ function docker_build {
         fi
         $PY3CMD ${DOCKER_SRC_DIR}/verify_licenses.py ${image_full_name}
     fi
-    if [ -f "verify.py" ]; then
-        echo "==========================="            
-        echo "Verifying docker image by running the python script verify.py within the docker image"
-        cat verify.py | docker run --rm -i ${image_full_name} python '-'
-    fi
+
+    for filename in `find . -name "*verify.py"`; do
```

Comment:
We're finding every verify py in the repo?

Reply:
no, we are finding every `verify.py` for the entire folder which the docker resides in. in our case it will find all the files that their name is anything_verify.py and that are inside py3-native folder and will run them on the container.